plugins/module_utils/aci.py

```diff
@@ -295,7 +301,7 @@ def login(self):
             url = "%(protocol)s://%(host)s:%(port)s/api/aaaLogin.json" % self.params
         else:
             url = "%(protocol)s://%(host)s/api/aaaLogin.json" % self.params
-        payload = {"aaaUser": {"attributes": {"name": self.params.get("username"), "pwd": self.params.get("password")}}}
+        payload = {"aaaUser": {"attributes": {"name": self.params.get("username", self.default_username), "pwd": self.params.get("password")}}}
```

Comment:
can we change this line from ----> payload = {"aaaUser": {"attributes": {"name": self.params.get("username", self.default_username), "pwd": self.params.get("password")}}}   ------->(to) ------> 
payload = {
                   "aaaUser": {
                        "attributes": {
                               "name": self.params.get("username", self.default_username), 
                               "pwd": self.params.get("password")
                          }
                }
         }